src/xcode/ENA/ENA/Source/Services/CoronaTestService/CoronaTestService.swift

```diff
@@ -554,8 +554,12 @@ class CoronaTestService: CoronaTestServiceProviding {
 		switch coronaTestType {
 		case .pcr:
 			pcrTest.value = nil
+			// updating badge count for home tab
```

Comment:
What do you think about to remove the comments, and instead we rename the `badgeWrapper` property?

```swift
// Old
private let badgeWrapper: HomeBadgeWrapper

// New
private let homeBadgeWrapper: HomeBadgeWrapper
```

Reply:
I was also thinking but the property was declared previously and need to refactored a lot then because it's used in several classes.

Reply:
ok then keep you comments